2019-07-03T09:25:19.254+0000 I CONTROL  [main] Automatically disabling TLS 1.0, to force-enable TLS 1.0 specify --sslDisabledProtocols 'none'
2019-07-03T09:25:19.258+0000 I CONTROL  [initandlisten] MongoDB starting : pid=1 port=30003 dbpath=/data/db 64-bit host=1de761af48a9
2019-07-03T09:25:19.261+0000 I CONTROL  [initandlisten] db version v4.0.10
2019-07-03T09:25:19.262+0000 I CONTROL  [initandlisten] git version: c389e7f69f637f7a1ac3cc9fae843b635f20b766
2019-07-03T09:25:19.263+0000 I CONTROL  [initandlisten] OpenSSL version: OpenSSL 1.0.2g  1 Mar 2016
2019-07-03T09:25:19.265+0000 I CONTROL  [initandlisten] allocator: tcmalloc
2019-07-03T09:25:19.269+0000 I CONTROL  [initandlisten] modules: none
2019-07-03T09:25:19.270+0000 I CONTROL  [initandlisten] build environment:
2019-07-03T09:25:19.271+0000 I CONTROL  [initandlisten]     distmod: ubuntu1604
2019-07-03T09:25:19.271+0000 I CONTROL  [initandlisten]     distarch: x86_64
2019-07-03T09:25:19.273+0000 I CONTROL  [initandlisten]     target_arch: x86_64
2019-07-03T09:25:19.274+0000 I CONTROL  [initandlisten] options: { net: { bindIpAll: true, port: 30003 }, replication: { replSet: "my-mongo-set" }, systemLog: { destination: "file", path: "mongo.log" } }
2019-07-03T09:25:19.276+0000 W STORAGE  [initandlisten] Detected unclean shutdown - /data/db/mongod.lock is not empty.
2019-07-03T09:25:19.278+0000 I STORAGE  [initandlisten] Detected data files in /data/db created by the 'wiredTiger' storage engine, so setting the active storage engine to 'wiredTiger'.
2019-07-03T09:25:19.278+0000 W STORAGE  [initandlisten] Recovering data from the last clean checkpoint.
2019-07-03T09:25:19.279+0000 I STORAGE  [initandlisten] 
2019-07-03T09:25:19.280+0000 I STORAGE  [initandlisten] ** WARNING: Using the XFS filesystem is strongly recommended with the WiredTiger storage engine
2019-07-03T09:25:19.280+0000 I STORAGE  [initandlisten] **          See http://dochub.mongodb.org/core/prodnotes-filesystem
2019-07-03T09:25:19.281+0000 I STORAGE  [initandlisten] wiredtiger_open config: create,cache_size=487M,session_max=20000,eviction=(threads_min=4,threads_max=4),config_base=false,statistics=(fast),log=(enabled=true,archive=true,path=journal,compressor=snappy),file_manager=(close_idle_time=100000),statistics_log=(wait=0),verbose=(recovery_progress),
2019-07-03T09:25:21.324+0000 I STORAGE  [initandlisten] WiredTiger message [1562145921:324103][1:0x7f7d935daa80], txn-recover: Main recovery loop: starting at 1/70784 to 2/256
2019-07-03T09:25:21.328+0000 I STORAGE  [initandlisten] WiredTiger message [1562145921:328324][1:0x7f7d935daa80], txn-recover: Recovering log 1 through 2
2019-07-03T09:25:21.590+0000 I STORAGE  [initandlisten] WiredTiger message [1562145921:590288][1:0x7f7d935daa80], file:sizeStorer.wt, txn-recover: Recovering log 2 through 2
2019-07-03T09:25:21.670+0000 I STORAGE  [initandlisten] WiredTiger message [1562145921:670896][1:0x7f7d935daa80], file:sizeStorer.wt, txn-recover: Set global recovery timestamp: 5d1c741d00000001
2019-07-03T09:25:21.891+0000 I RECOVERY [initandlisten] WiredTiger recoveryTimestamp. Ts: Timestamp(1562145821, 1)
2019-07-03T09:25:21.893+0000 I STORAGE  [initandlisten] Triggering the first stable checkpoint. Initial Data: Timestamp(1562145821, 1) PrevStable: Timestamp(0, 0) CurrStable: Timestamp(1562145821, 1)
2019-07-03T09:25:22.047+0000 I STORAGE  [initandlisten] Starting OplogTruncaterThread local.oplog.rs
2019-07-03T09:25:22.048+0000 I STORAGE  [initandlisten] The size storer reports that the oplog contains 37 records totaling to 4936 bytes
2019-07-03T09:25:22.049+0000 I STORAGE  [initandlisten] Scanning the oplog to determine where to place markers for truncation
2019-07-03T09:25:22.147+0000 I CONTROL  [initandlisten] 
2019-07-03T09:25:22.148+0000 I CONTROL  [initandlisten] ** WARNING: Access control is not enabled for the database.
2019-07-03T09:25:22.149+0000 I CONTROL  [initandlisten] **          Read and write access to data and configuration is unrestricted.
2019-07-03T09:25:22.150+0000 I CONTROL  [initandlisten] 
2019-07-03T09:25:22.259+0000 I STORAGE  [initandlisten] Dropping unknown ident: collection-25--4945751950579637342
2019-07-03T09:25:22.274+0000 I STORAGE  [initandlisten] Dropping unknown ident: index-26--4945751950579637342
2019-07-03T09:25:22.279+0000 I STORAGE  [initandlisten] Dropping unknown ident: index-27--4945751950579637342
2019-07-03T09:25:22.311+0000 I FTDC     [initandlisten] Initializing full-time diagnostic data capture with directory '/data/db/diagnostic.data'
2019-07-03T09:25:22.322+0000 I REPL     [initandlisten] Rollback ID is 1
2019-07-03T09:25:22.339+0000 I REPL     [initandlisten] Recovering from stable timestamp: Timestamp(1562145821, 1) (top of oplog: { ts: Timestamp(1562145867, 3), t: 1 }, appliedThrough: { ts: Timestamp(1562145821, 1), t: 1 }, TruncateAfter: Timestamp(0, 0))
2019-07-03T09:25:22.342+0000 I REPL     [initandlisten] Starting recovery oplog application at the stable timestamp: Timestamp(1562145821, 1)
2019-07-03T09:25:22.342+0000 I REPL     [initandlisten] Replaying stored operations from { : Timestamp(1562145821, 1) } (exclusive) to { : Timestamp(1562145867, 3) } (inclusive).
2019-07-03T09:25:22.356+0000 I STORAGE  [repl writer worker 14] createCollection: config.system.sessions with provided UUID: 09370cb4-da8b-4b69-ae95-b826dfd21bdb
2019-07-03T09:25:22.468+0000 I REPL     [repl writer worker 14] applied op: command { ts: Timestamp(1562145867, 1), t: 1, h: -2155606415205593510, v: 2, op: "c", ns: "config.$cmd", ui: UUID("09370cb4-da8b-4b69-ae95-b826dfd21bdb"), wall: new Date(1562145867210), o: { create: "system.sessions", idIndex: { v: 2, key: { _id: 1 }, name: "_id_", ns: "config.system.sessions" } } }, took 112ms
2019-07-03T09:25:22.488+0000 I INDEX    [repl writer worker 1] build index on: config.system.sessions properties: { v: 2, key: { lastUse: 1 }, name: "lsidTTLIndex", expireAfterSeconds: 1800, ns: "config.system.sessions" }
2019-07-03T09:25:22.488+0000 I INDEX    [repl writer worker 1] 	 building index using bulk method; build may temporarily use up to 500 megabytes of RAM
2019-07-03T09:25:22.490+0000 I INDEX    [repl writer worker 1] build index done.  scanned 0 total records. 0 secs
2019-07-03T09:25:22.494+0000 I NETWORK  [initandlisten] waiting for connections on port 30003
2019-07-03T09:25:22.495+0000 I CONTROL  [LogicalSessionCacheRefresh] Sessions collection is not set up; waiting until next sessions refresh interval: Replication has not yet been configured
2019-07-03T09:25:22.542+0000 I REPL     [replexec-0] New replica set config in use: { _id: "my-mongo-set", version: 1, protocolVersion: 1, writeConcernMajorityJournalDefault: true, members: [ { _id: 1, host: "mongo1:30001", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 2, host: "mongo2:30002", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 }, { _id: 3, host: "mongo3:30003", arbiterOnly: false, buildIndexes: true, hidden: false, priority: 1.0, tags: {}, slaveDelay: 0, votes: 1 } ], settings: { chainingAllowed: true, heartbeatIntervalMillis: 2000, heartbeatTimeoutSecs: 10, electionTimeoutMillis: 10000, catchUpTimeoutMillis: -1, catchUpTakeoverDelayMillis: 30000, getLastErrorModes: {}, getLastErrorDefaults: { w: 1, wtimeout: 0 }, replicaSetId: ObjectId('5d1c73211e7bf786f0a5ba36') } }
2019-07-03T09:25:22.543+0000 I REPL     [replexec-0] This node is mongo3:30003 in the config
2019-07-03T09:25:22.544+0000 I REPL     [replexec-0] transition to STARTUP2 from STARTUP
2019-07-03T09:25:22.544+0000 I REPL     [replexec-0] Starting replication storage threads
2019-07-03T09:25:22.547+0000 I ASIO     [Replication] Connecting to mongo1:30001
2019-07-03T09:25:22.548+0000 I ASIO     [Replication] Connecting to mongo2:30002
2019-07-03T09:25:22.547+0000 I REPL     [replexec-0] transition to RECOVERING from STARTUP2
2019-07-03T09:25:22.549+0000 I REPL     [replexec-0] Starting replication fetcher thread
2019-07-03T09:25:22.550+0000 I REPL     [replexec-0] Starting replication applier thread
2019-07-03T09:25:22.550+0000 I ASIO     [Replication] Failed to connect to mongo1:30001 - HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.552+0000 I CONNPOOL [Replication] Dropping all pooled connections to mongo1:30001 due to HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.551+0000 I REPL     [rsSync-0] Starting oplog application
2019-07-03T09:25:22.551+0000 I REPL     [replexec-0] Starting replication reporter thread
2019-07-03T09:25:22.554+0000 I REPL_HB  [replexec-1] Error in heartbeat (requestId: 1) to mongo1:30001, response status: HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.557+0000 I REPL     [rsSync-0] transition to SECONDARY from RECOVERING
2019-07-03T09:25:22.558+0000 I REPL     [rsSync-0] Resetting sync source to empty, which was :27017
2019-07-03T09:25:22.558+0000 I ASIO     [Replication] Connecting to mongo1:30001
2019-07-03T09:25:22.562+0000 I REPL     [replexec-0] Member mongo2:30002 is now in state SECONDARY
2019-07-03T09:25:22.573+0000 I ASIO     [Replication] Failed to connect to mongo1:30001 - HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.573+0000 I CONNPOOL [Replication] Dropping all pooled connections to mongo1:30001 due to HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.574+0000 I REPL_HB  [replexec-1] Error in heartbeat (requestId: 3) to mongo1:30001, response status: HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.574+0000 I ASIO     [Replication] Connecting to mongo1:30001
2019-07-03T09:25:22.576+0000 I ASIO     [Replication] Failed to connect to mongo1:30001 - HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.576+0000 I CONNPOOL [Replication] Dropping all pooled connections to mongo1:30001 due to HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.577+0000 I REPL_HB  [replexec-0] Error in heartbeat (requestId: 4) to mongo1:30001, response status: HostUnreachable: Error connecting to mongo1:30001 (172.22.0.4:30001) :: caused by :: Connection refused
2019-07-03T09:25:22.577+0000 I REPL     [replexec-0] Member mongo1:30001 is now in state RS_DOWN
2019-07-03T09:25:22.895+0000 I NETWORK  [listener] connection accepted from 172.22.0.4:59986 #3 (1 connection now open)
2019-07-03T09:25:22.897+0000 I NETWORK  [conn3] end connection 172.22.0.4:59986 (0 connections now open)
2019-07-03T09:25:22.914+0000 I NETWORK  [listener] connection accepted from 172.22.0.4:59990 #4 (1 connection now open)
2019-07-03T09:25:22.915+0000 I NETWORK  [conn4] received client metadata from 172.22.0.4:59990 conn4: { driver: { name: "NetworkInterfaceTL", version: "4.0.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2019-07-03T09:25:22.985+0000 I NETWORK  [listener] connection accepted from 172.22.0.3:60046 #5 (2 connections now open)
2019-07-03T09:25:22.987+0000 I NETWORK  [conn5] received client metadata from 172.22.0.3:60046 conn5: { driver: { name: "NetworkInterfaceTL", version: "4.0.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2019-07-03T09:25:23.077+0000 I ASIO     [Replication] Connecting to mongo1:30001
2019-07-03T09:25:23.098+0000 I FTDC     [ftdc] Unclean full-time diagnostic data capture shutdown detected, found interim file, some metrics may have been lost. OK
2019-07-03T09:25:23.126+0000 I REPL     [replexec-1] Member mongo1:30001 is now in state SECONDARY
2019-07-03T09:25:27.120+0000 I NETWORK  [listener] connection accepted from 172.22.0.4:60004 #7 (3 connections now open)
2019-07-03T09:25:27.125+0000 I NETWORK  [conn7] received client metadata from 172.22.0.4:60004 conn7: { driver: { name: "MongoDB Internal Client", version: "4.0.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2019-07-03T09:25:33.193+0000 I REPL     [replexec-0] Member mongo2:30002 is now in state PRIMARY
2019-07-03T09:25:34.720+0000 I NETWORK  [conn7] end connection 172.22.0.4:60004 (2 connections now open)
2019-07-03T09:25:35.574+0000 I REPL     [rsBackgroundSync] sync source candidate: mongo2:30002
2019-07-03T09:25:35.577+0000 I ASIO     [RS] Connecting to mongo2:30002
2019-07-03T09:25:35.588+0000 I REPL     [rsBackgroundSync] Changed sync source from empty to mongo2:30002
2019-07-03T09:25:35.589+0000 I ASIO     [RS] Connecting to mongo2:30002
2019-07-03T09:30:22.497+0000 I NETWORK  [LogicalSessionCacheRefresh] Starting new replica set monitor for my-mongo-set/mongo1:30001,mongo2:30002,mongo3:30003
2019-07-03T09:30:22.504+0000 I NETWORK  [listener] connection accepted from 172.22.0.2:56556 #11 (3 connections now open)
2019-07-03T09:30:22.505+0000 I NETWORK  [conn11] received client metadata from 172.22.0.2:56556 conn11: { driver: { name: "MongoDB Internal Client", version: "4.0.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2019-07-03T09:30:22.508+0000 I NETWORK  [ReplicaSetMonitor-TaskExecutor] Successfully connected to mongo3:30003 (1 connections now open to mongo3:30003 with a 5 second timeout)
2019-07-03T09:30:22.510+0000 I NETWORK  [LogicalSessionCacheRefresh] Successfully connected to mongo2:30002 (1 connections now open to mongo2:30002 with a 5 second timeout)
2019-07-03T09:30:22.515+0000 I NETWORK  [ReplicaSetMonitor-TaskExecutor] Successfully connected to mongo1:30001 (1 connections now open to mongo1:30001 with a 5 second timeout)
2019-07-03T09:30:22.516+0000 I NETWORK  [LogicalSessionCacheRefresh] Successfully connected to mongo2:30002 (1 connections now open to mongo2:30002 with a 0 second timeout)
2019-07-03T09:30:22.517+0000 I NETWORK  [LogicalSessionCacheRefresh] Starting new replica set monitor for my-mongo-set/mongo1:30001,mongo2:30002,mongo3:30003
2019-07-03T09:30:22.519+0000 I NETWORK  [LogicalSessionCacheRefresh] Starting new replica set monitor for my-mongo-set/mongo1:30001,mongo2:30002,mongo3:30003
2019-07-03T09:30:22.870+0000 I NETWORK  [listener] connection accepted from 172.22.0.4:60068 #15 (4 connections now open)
2019-07-03T09:30:22.871+0000 I NETWORK  [conn15] received client metadata from 172.22.0.4:60068 conn15: { driver: { name: "MongoDB Internal Client", version: "4.0.10" }, os: { type: "Linux", name: "Ubuntu", architecture: "x86_64", version: "16.04" } }
2019-07-03T09:35:02.416+0000 I NETWORK  [listener] connection accepted from 172.22.0.5:43456 #16 (5 connections now open)
2019-07-03T09:35:02.432+0000 I NETWORK  [conn16] received client metadata from 172.22.0.5:43456 conn16: { driver: { name: "nodejs", version: "3.2.7" }, os: { type: "Linux", name: "linux", architecture: "x64", version: "4.9.125-linuxkit" }, platform: "Node.js v7.10.1, LE, mongodb-core: 3.2.7" }
2019-07-03T09:35:02.569+0000 I STORAGE  [repl writer worker 3] createCollection: test.tasks with provided UUID: 9be2c1bd-8090-4455-aa48-9c09c191ef6e
